searchMatrix.py

```diff
@@ -0,0 +1,29 @@
+# Brute Force Solution

```

Comment:
Submit the optimised solution for this problem where you do not need to copy the matrix elements in new list